frb_codegen/src/library/codegen/generator/wire/rust/spec_generator/misc/mod.rs

```diff
@@ -114,20 +113,8 @@ use flutter_rust_bridge::for_generated::byteorder::{NativeEndian, WriteBytesExt,
     })
 }
 
-fn generate_wrapper_struct(ty: &IrType, context: WireRustGeneratorContext) -> Option<String> {
-    // the generated wrapper structs need to be public for the StreamSinkTrait impl to work
-    WireRustGenerator::new(ty.clone(), context)
-        .wrapper_struct_name()
-        .map(|wrapper_struct_name| {
-            format!(
-                r###"
-                #[derive(Clone)]
-                pub struct {}({});
-                "###,
-                wrapper_struct_name,
-                ty.rust_api_type(),
-            )
-        })
+fn generate_wrapper_struct(_ty: &IrType, _context: WireRustGeneratorContext) -> Option<String> {
+    None
```

Comment:
nit: if it is none, iirc we may be able to remove this function since it has default implementation